torch/_weights_only_unpickler.py

```diff
@@ -331,7 +339,10 @@ def load(self):
                 cls = self.stack.pop()
                 if cls is torch.nn.Parameter:
                     self.append(torch.nn.Parameter(*args))
-                elif cls in _get_user_allowed_globals().values():
+                elif (
+                    cls in _get_user_allowed_globals().values()
+                    or cls in _get_allowed_globals().values()
```

Comment:
Technically perhaps we could be more restrictive here and just use the set of classes that need to go through this elif statement instead of `_get_allowed_globals.values()`, since the `REDUCE` instruction already allows anything in `_get_allowed_globals().values()` to be called, so I think this is fine as is